cellpack/autopack/ingredient/__init__.py

```diff
@@ -7,3 +8,17 @@
 from .Ingredient import Ingredient  # noqa: F401
 
 __all__ = ["grow", "multi_cylinder", "multi_sphere", "single_cube", "single_sphere"]
+
+
+type_to_class_map = {
+    INGREDIENT_TYPE.SINGLE_SPHERE: SingleSphereIngr,
+    INGREDIENT_TYPE.MULTI_SPHERE: MultiSphereIngr,
+    INGREDIENT_TYPE.SINGLE_CUBE: SingleCubeIngr,
+    INGREDIENT_TYPE.SINGLE_CYLINDER: SingleCylinderIngr,
+    INGREDIENT_TYPE.MULTI_CYLINDER: MultiCylindersIngr,
+    INGREDIENT_TYPE.GROW: GrowIngredient,
+}
+
+
+def get_ingredient_class(ingredient_type):
+    return type_to_class_map[ingredient_type]
```

Comment:
I thought we might want to do other checks here, which is why I made this a function instead of just using the map directly